src/main/scala/rps/controller/GameController.scala

```diff
@@ -1,40 +1,39 @@
-package rps.controller
+package rps.controller.GameController
```

Comment:
in general, it doesn't make much sense to have a package named `GameController`.

I would just leave the `GameController` (or `GameApi`) implementation inside the `rps.controller` package.